### ISSUE TITLE
change_node.jstree not working

### ISSUE THREAD

**Issue by srkunze** [deitch/jstree-grid#163]
Hi, I am using the latest version of jstree-grid and tried to get change_node.jstree working. I used the example described here https://github.com/deitch/jstree-grid/blob/master/README.md

```
var node = $("#jstree").jstree(true).get_node("my_node");
node.data.value = 25;
node.trigger("change_node.jstree");
```

It says that "trigger" is not defined at node.

**Comment by fcpana** [comment]
I had the same problem, the following worked for me:

$('#jstree').jstree(true).trigger('change_node.jstree', node);

**Comment by deitch** [comment]
It did work at some point; I think the API must have changed. I will update the README. @fcpana thank you.

**Comment by deitch** [comment]
Fixed in commit df3a722daa4422309b76a5b4e6caa0d11a8b18ac
